test/.coveragerc

```diff
@@ -1,6 +1,20 @@
 [report]
-omit =
-    */python?.?/*
-    */site-packages/nose/*
+;fail_under = 100
+exclude_lines =
```

Comment:
Yes, and they suit our needs. Those examples are from the Coverage module author. They are what he suggests as most common parameters for coverage